package.json

```diff
@@ -62,5 +62,11 @@
 		"@wordpress/babel-preset-default": "^6.4.1",
 		"lodash": "^4.17.21",
 		"wp-textdomain": "1.0.1"
+	},
+	"pnpm": {
```

Comment:
Since this impacts the entire mono-repo, I wonder if we should loop in someone from Atlas so they at least are aware of it? Perhaps @psealock or @samueljseay has thoughts?